contrib/debian/README.md

```diff
@@ -15,7 +15,7 @@ To install:
 
 If you build yourself, you will either need to modify the paths in
 the .desktop file or copy or symlink your dash-qt binary to `/usr/bin`
-and the `../../share/pixmaps/dash128.png` to `/usr/share/pixmaps`
+and the `../../share/pixmaps/bitcoin128.png` to `/usr/share/pixmaps`
```

Comment:
Errr... I'm not sure if it makes sense to enable ppa builds which will conflict with some existing one... how about fixing /contrib/debian/dash-qt.install instead (at least that's the only reference I found) and renaming bitcoin128.png to dash128.png?